### ISSUE TITLE
Some issues are not shown

### ISSUE THREAD

**Issue by vikeri** [drapanjanas/re-natal#44]
_reagent6, iOS, Mac_

TLDR: Some RN errors are not reported anywhere

All of a sudden the hot code reload fails and no more changes are pushed. That usually means that I screwed something up. If I reload the emulator I just get `Waiting for Figwheel to load files`. The figwheel REPL shows nothing and a `(reset-autobuild)` successfully recompiles the project.

It is like these issues fall between compile issues shown by figwheel and the red screen of RN. I have not  got around to make an MWE but an example issue is when I accidentally ran the function `r/adapt-react-class` twice on an object.

Not sure if this issue is related to re-natal but I never experienced them in React Web.

**Comment by vikeri** [comment]
Another kind of issue that will present this behaviour is when `(dispatch-sync [:initialize-db])` fails.

**Comment by drapanjanas** [comment]
Yep, I could reproduce it and fixed. The problem was in case of debugging in chrome erors thrown by importScripts function were not logged at all. No errors logged - no red screen.
I also removed some error messages which did not look very useful.

**Comment by vikeri** [comment]
Fix confirmed, great job!
